Documents/public-api.md

```diff
@@ -0,0 +1,34 @@
+# HMDA Platform Public API
+
+This documenatation describes de public HMDA Platform HTTP API
+
+## Institutions
+
+### Search
+
+* `/institutions?domain=<domain>`
+
+   * `GET` - Returns a list of institutions filtered by their email domain. If none are found, an HTTP 404 error code (not found) is returned
```

Comment:
I don't have a real preference. Can we provide a message in the case of no results?